batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/constants/EmployeeDataConstants.java

```diff
@@ -0,0 +1,11 @@
+package com.lftechnology.batch7crud.constants;
+
+public class EmployeeDataConstants {
+    public static final String FIRST_NAME = "firstName";
+    public static final String LAST_NAME = "lastName";
+    public static final String ADDRESS = "address";
+    public static final String DEPARTMENT = "department";
+    
+    private EmployeeDataConstants(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)